src/Routing/Route/Route.php

```diff
@@ -479,6 +479,11 @@ public function parse($url, $method = '')
             $route['_ext'] = $ext;
         }
 
+        // pass the name if set
+        if (isset($this->options['_name'])) {
+            $route['_name'] = $this->options['_name'];
+        }
```

Comment:
Any chance tests could be updated to account for this additional key?

Reply:
Hey, ofcourse
I will add some tests tomorrow

Reply:
I think this key might need to be removed by `Router::reverse()` in this section https://github.com/cakephp/cakephp/blob/e3c3594e24bfc83bfbcd772b19789cdd0557f6c1/src/Routing/Router.php#L738-L754

Or the `_name` parameter will impact the results.